composer.json

```diff
@@ -19,20 +19,20 @@
 		}
 	],
 	"require": {
-		"php": ">=7.4",
+		"php": ">=8.1",
```

Comment:
Why this? For cake4?

Reply:
Basically because if we remove `cakephp-adapter` and add it to conflicts, composer adds `symfony/http-client` and it requires php >= 8.1 (version 6) or php >= 8.2 (version 7). 
```
 Problem 1
    - Root composer.json requires symfony/http-client ^7.0 -> satisfiable by symfony/http-client[v7.0.0, v7.0.2, v7.0.3].
    - symfony/http-client[v7.0.0, ..., v7.0.3] require php >=8.2 -> your php version (7.4.33) does not satisfy that requirement.
```

Then if we remove the conflict it fails as well:

```
 Problem 1
    - dereuromark/cakephp-geo dev-master conflicts with php-http/cakephp-adapter 0.3.3.
    - dereuromark/cakephp-geo dev-master conflicts with php-http/cakephp-adapter 0.3.2.
    - dereuromark/cakephp-geo dev-master conflicts with php-http/cakephp-adapter 0.3.1.
    - dereuromark/cakephp-geo dev-master conflicts with php-http/cakephp-adapter 0.3.0.
    - php-http/cakephp-adapter 0.1.0 requires cakephp/cakephp ^3.1 -> found cakephp/cakephp[3.1.0, ..., 3.10.5] but it conflicts with your root composer.json require (^4.2.0).
    - php-http/cakephp-adapter 0.2.0 requires cakephp/cakephp ^3.4.12 -> found cakephp/cakephp[3.4.12, ..., 3.10.5] but it conflicts with your root composer.json require (^4.2.0).
    - php-http/cakephp-adapter 0.4.0 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
    - dereuromark/cakephp-geo is present at version dev-master and cannot be modified by Composer
    - Root composer.json requires php-http/cakephp-adapter * -> satisfiable by php-http/cakephp-adapter[0.1.0, ..., 0.4.0].
```

So I was not able to make it work with php 7.4 and then I thought that since php 7.4 ended support 1 year, 2 months ago it would be good to increase the version and, if somebody needs to use cakephp 7.4, they can use any previous version. But I am open to suggestions of course :)

Reply:
Fair enough I guess.